jwst/skymatch/region.py

```diff
@@ -1,8 +1,9 @@
 """
 Polygon filling algorithm.
 
-:Authors: Nadezhda Dencheva, Mihai Cara (contact: help@stsci.edu)
+:Authors: Nadezhda Dencheva, Mihai Cara
 
+:License: :doc:`LICENSE`
```

Comment:
I would remove the authors and license to be consistent with the rest of the code in jwst.